mrp_production_capacity/models/mrp.py

```diff
@@ -38,10 +24,14 @@ class MrpRoutingWorkcenter(models.Model):
 class MrpProduction(models.Model):
     _inherit = 'mrp.production'
 
+    show_split_button = fields.Boolean('Show split button', default=False)
```

Comment:
Y qué pasa si tú creas la OF por código? Ya no se cambia ese dato. Por eso debe ser un calculado.